utils/oauthTokensUtil.ts

```diff
@@ -0,0 +1,105 @@
+import { Response } from "express";
+import { v4 as uuidv4 } from 'uuid';
+import { redisClient } from '../db/redis';
+
+const accessTokenExpiration = 900;
```

Comment:
You can add this consts in env file 
```suggestion
const accessTokenExpiration = 900;
const refreshTokenExpiration = 21600;
```